src/coreclr/tools/r2rdump/TextDumper.cs

```diff
@@ -561,13 +561,55 @@ internal override void DumpFixupStats()
                     Count = group.Count()
                 }).OrderByDescending(x => x.Count);
 
-            Console.WriteLine($"                      Fixup | Count");
+            /* Runtime Repo GH Issue #49249:
+             *
+             * In order to format the fixup counts results table, we need to
+             * know beforehand the size of each column. */
+
+            int longestFixupKind = 0;
```

Comment:
Non-blocking nit - you could rename this variable to "fixupPadding", initialize it to 5 and delete the line 595. If you're more comfortable with your formulation, feel free to keep it.